scania-digital-design-system/sdds
# fix(dropdown): add placeholder for label inside

**Describe pull-request**
Dropdown with label inside is not rendering the placeholder text.

**Solving issue**
Github issue https://github.com/scania-digital-design-system/sdds/issues/440 and Azure ticket [AB#2321](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2321)

**How to test**
Add description how to test if possible

Go to storybook
Check dropdown label inside and see if it looks correct
